docs/01-app/04-api-reference/04-functions/use-link-status.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: useLinkStatus
+description: API Reference for the useLinkStatus hook.
+---
+
+`useLinkStatus` is a **Client Component** hook that lets you track the loading state of a `Link` component during navigation. It's particularly useful when you want to show loading indicators during page transitions, especially when prefetching is disabled.
```

Comment:
```suggestion
`useLinkStatus` is a **Client Component** hook that lets you track the loading state of a `Link` component during navigation. It can be used to show loading indicators during page transitions, especially when [prefetching](/docs/app/building-your-application/routing/linking-and-navigating#2-prefetching) is disabled.
```